ONSdigital/eq-survey-runner
# Pattern Library Foundation

## Changes
- Basic system to walk directory of components and generate documentation.
- Added routes for pattern library.
- Added some content to typography, grid system and colours.
- Added styling for various things.
## How to test
- `npm run build`
- navigate to /patterns
- confirm pages load correctly w/ styling
